transformer.py

```diff
@@ -206,12 +220,14 @@ def check_continue(transformer: transformer_class.Transformer, check_md: dict, t
             break
 
     # Return the appropriate result
-    return (0) if found_file else (-1, "Unable to find an image file to work with")
+    return 0 if found_file else (-1, "Unable to find an image file to work with")
```

Comment:
This will now return an integer (zero) or a tuple, when it should return a tuple (see return type declaration of function signature)